devicetypes/smartthings/zigbee-window-shade-battery.src/zigbee-window-shade-battery.groovy

```diff
@@ -121,18 +124,23 @@ def parse(String description) {
 			levelEventHandler(valueInt)
 		} else if (reportsBatteryPercentage() && descMap?.clusterInt == zigbee.POWER_CONFIGURATION_CLUSTER && zigbee.convertHexToInt(descMap?.attrId) == BATTERY_PERCENTAGE_REMAINING && descMap.value) {
 			def batteryLevel = zigbee.convertHexToInt(descMap.value)
+
 			batteryPercentageEventHandler(batteryLevel)
 		}
 	}
 }
 
 def levelEventHandler(currentLevel) {
-	def lastLevel = device.currentValue("level")
+	def lastLevel = device.currentValue("shadeLevel")
+
 	log.debug "levelEventHandle - currentLevel: ${currentLevel} lastLevel: ${lastLevel}"
+
 	if (lastLevel == "undefined" || currentLevel == lastLevel) { //Ignore invalid reports
 		log.debug "Ignore invalid reports"
 	} else {
+		sendEvent(name: "shadeLevel", value: currentLevel)
 		sendEvent(name: "level", value: currentLevel)
```

Comment:
I think we may want to force `displayed` false here so it only shows `shadeLevel` in history